gt/main.c

```diff
@@ -1427,11 +1464,14 @@ process_gt_cmd(struct gt_cmd_entry *entry, struct gt_instance *instance)
 		if ((luaL_loadbuffer(instance->lua_state,
 				entry->u.bc.lua_bytecode, entry->u.bc.len,
 				"incremental_update_of_gt_lua_state") != 0) ||
-				(lua_pcall(instance->lua_state, 0, 0, 0) != 0)) {
+				(lua_pcall(instance->lua_state, 0,
+					entry->u.bc.is_returned, 0) != 0)) {
```

Comment:
Add `!!` before `entry->u.bc.is_returned` to make sure the expression is either 0 or 1 and not 2, 10, 11...